### PR TITLE
Update legacy Makefile to restraint metadata (part 1)

### PR DESCRIPTION
**Pull request by ftjahjad**
update to metadata:
* cki_lib  library
* cpu tests
* firmware tests
* ipmi tests
* jvm tests
* kdump tests
* pciutils tests

### CODE REVIEW

**Comment by ftjahjad** [comment]
@cki-bot , please test

**Comment by ftjahjad** [comment]
@cki-bot  please test again

File: cki_lib/metadata [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[General]
+name=cki_lib
+description=Common CKI library for beaker testing
+license=GPLv2
+confidential=no
+destructive=no
+
+[restraint]
+entry_point=echo nothing > /dev/null
```
[the remark this answers is not in the document]
Comment (ftjahjad) [reply cki_lib/metadata:9]: This is just library,

File: cpu/die/metadata [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+name=cpu/die 
+owner=David Arcari <darcari@redhat.com>
+description=cpu die test
+license=GPLv2
+confidential=no
+destructive=no
+
+[restraint]
+entry_point=bash ./runtest.sh
+softDependencies=python2-lxml;python3-lxml
```
[the remark this answers is not in the document]
Comment (ftjahjad) [reply cpu/die/metadata:11]: restraint will not stop, if softdeps failing to install.
on legacy Makefile, Requires acts as softdependencies in restraint

File: pciutils/sanity-smoke/metadata [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+owner=William Gomeringer <willg@redhat.com>
+description=test whether lspci works
+license=GPLv2
+confidential=no
+destructive=no
+
+[restraint]
+entry_point=bash ./runtest.sh
+dependencies=pciutils
+softDependencies=kernel-modules-extra;python2-lxml;python3-lxml
```
[the remark this answers is not in the document]
Comment (ftjahjad) [reply pciutils/sanity-smoke/metadata:12]: That would be possible , after we are done on converting from legacy Makefile to metadata (restraint) first :-D.

File: firmware/include/metadata [hunk 4]
```diff
@@ -0,0 +1,13 @@
+[General]
+name=firmware/include
+owner=Erico Nunes <ernunes@redhat.com
```
Comment (rasibley) [inline firmware/include/metadata:3]: One minor nitpick, missing the` >` here

File: ipmi/stress/driver/metadata [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+name=ipmi/stress/driver
+owner=Rachel Sibley <rasibley@redhat.com>
+description=IPMI driver load stress test
+license=GPLv2
+confidential=no
+destructive=no
+
+[restraint]
+entry_point=bash ./runtest.sh
+dependencies=dmidecode
```
Comment (rasibley) [inline ipmi/stress/driver/metadata:11]: Please move dmidecode as a soft dependency, it's not supported on ppc64le, I have this built into the test, you will see it's aborting on this arch.

Comment (ftjahjad) [reply ipmi/stress/driver/metadata]: move "dmidecode" ipmi metadata tests to softDependencies

**Pull request merged**